dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Inventory/README.md

```diff
@@ -0,0 +1,3 @@
+# Magento 2 Acceptance Tests
```

Comment:
Based on our standard we call our tests/packages Functional.
Example
https://github.com/magento/magento2ce/tree/2.3-develop/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Authorization

Reply:
OOOPS! I copied the "Integration" module which used the words "Acceptance". I'll update my new modules.

Reply:
Changes committed!